packages/metadata/src/metadata-service.ts

```diff
@@ -45,10 +45,10 @@ function getActionName(name: Path): Path {
 
 /**
  *
- * @param metadata
- * @param actionNames
- * @param namespaces
- * @param metadataElements
+ * @param metadata metadata element map
+ * @param actionNames string map
```

Comment:
This description doesn't seem to bring much value.

Reply:
updated